lambda/getBasicInformationLambda.ts

```diff
@@ -81,23 +92,38 @@ export const handler = async (
 			await queueJob({
 				QueueUrl: validIssuers[issuer.issuerIdentifierNumber] as string,
 				sqs,
-			})({ payload: iccid, deduplicationId: iccid })
+			})({ payload: { iccid }, deduplicationId: iccid })
 			return res(toStatusCode[ErrorType.Conflict], { expires: 60 })()
 		}
 		//SIM not existing
 		return res(toStatusCode[ErrorType.EntityNotFound], { expires: 60 })()
 	}
-	//Case 2: old data in DynamoDB (> 5min)
-	const timeStampFromDB = maybeSimDetails.sim.timestamp
-	const isOld = olderThan5min(timeStampFromDB)
+	const timeStampFromDB = maybeSimDetails.sim.ts
+	const isOld = olderThan5min({ timeStampFromDB })
 	if (isOld == true) {
 		await queueJob({
 			QueueUrl: validIssuers[issuer.issuerIdentifierNumber] as string,
 			sqs,
-		})({ payload: iccid, deduplicationId: iccid })
-		return res(200, { expires: 300 })(maybeSimDetails.sim)
+		})({ payload: { iccid }, deduplicationId: iccid })
+	}
+	const timeSpans = HistoricalDataTimeSpans[timeSpan]
+	if (timeSpans !== undefined) {
+		const result = await getBinIntervalFunc({
+			binIntervalMinutes: timeSpans.binIntervalMinutes,
+			durationHours: timeSpans.durationHours,
+			iccid,
+		})
+		const measurements = []
+		for (const measurement of result) {
+			measurements.push({
+				ts: measurement.time,
+				usedBytes: measurement['measure_value::double'],
+			})
+		}
```

Comment:
```suggestion
		const measurements = result.map(measurement => (({
				ts: measurement.time,
				usedBytes: measurement['measure_value::double'],
			}))
```

Can be made more compact using a map.